### PR TITLE
PCT-1812: Several changes to make go vet happy

### PR DESCRIPTION
**Pull request by mpaolino**

### CODE REVIEW

**Comment by cfsalguero** [comment]
:+1: 
I like explicit field in structs initialization

File: test/sync.go [hunk 1]
```diff
@@ -278,7 +277,6 @@ func WaitStatusPrefix(timeout int, r pct.StatusReporter, proc string, state stri
 			}
 		}
 	}
-	return false
```
Comment (daniel-nichter) [inline test/sync.go]: This is correct? Seems like explicit `return false` is what we want.

**Comment by daniel-nichter** [comment]
Apart from one question, :+1:

File: test/sync.go (hunk 1 above)
Comment (mpaolino) [inline test/sync.go]: its dead code, never reached

**Comment by arvenil** [comment]
:+1:

**Comment by mpaolino** [comment]
@daniel-nichter any further questions? Can I merge this?

**Pull request merged**